src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,8 @@
+spring:
```

Comment:
we dont need this, for now we can remove it

Reply:
removed